src/main/java/org/jenkinsci/plugins/parallel_test_executor/ParallelTestExecutor.java

```diff
@@ -143,70 +144,73 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
         return true;
     }
 
-    static List<InclusionExclusionPattern> findTestSplits(Parallelism parallelism, Run<?,?> build, TaskListener listener, boolean generateInclusions) {
+    static List<InclusionExclusionPattern> findTestSplits(Parallelism parallelism, Run<?,?> build, TaskListener listener,
+                                                          boolean generateInclusions,
+                                                          @CheckForNull final PreviousTestResultLookup lookup) {
         TestResult tr = findPreviousTestResult(build, listener);
         if (tr == null) {
             listener.getLogger().println("No record available, so executing everything in one place");
             return Collections.singletonList(new InclusionExclusionPattern(Collections.<String>emptyList(), false));
-        } else {
-
-            Map<String/*fully qualified class name*/, TestClass> data = new TreeMap<>();
```

Comment:
[Diff for review](https://github.com/jenkinsci/parallel-test-executor-plugin/pull/31/files?w=1). Recommend avoiding refactorings that force reindentation unless necessary for clarity.